src/renderers/dom/shared/wrappers/__tests__/ReactDOMInput-test.js

```diff
@@ -1003,7 +1022,7 @@ describe('ReactDOMInput', () => {
     );
     ReactDOM.render(<input type="radio" value="value" />, container);
     expectDev(console.error.calls.argsFor(0)[0]).toContain(
-      'A component is changing a controlled input of type radio to be uncontrolled. ' +
+      'Warning: A component is changing a controlled input of type radio to be uncontrolled. ' +
         'Input elements should not switch from controlled to uncontrolled (or vice versa). ' +
         'Decide between using a controlled or uncontrolled input ' +
         'element for the lifetime of the component. More info: https://fb.me/react-controlled-components',
```

Comment:
nit: Curious why we're checking the stack portion in all of the above tests but not this one